src/frontend/views/Youtube/YoutubeTemplate.tsx

```diff
@@ -93,8 +93,9 @@ export function YoutubeTemplate({
 
       {status === 'urlReady' && (
         <p className={styles.buttonsLine}>
-          <a className={styles.ctaButton} href={authUrl}>
-            Start Credential process
+          <a className={styles.signinButton} href={authUrl}>
+            Sign-in with
+            <span className={styles.youtubeBrandLogo}></span>
```

Comment:
```suggestion
```

Reply:
It's good practice to avoid empty DOM elements if you can do the same thing using just CSS.